objectstorage/transfer/multipart_manifest.go

```diff
@@ -17,14 +17,15 @@ type multipartManifest struct {
 }
 
 type uploadPart struct {
-	size     int64
-	offset   int64
-	partBody []byte
-	partNum  int
-	hash     *string
-	opcMD5   *string
-	etag     *string
-	err      error
+	size       int64
```

Comment:
looks like lots of formatting diffs. could you please run make test locally?

Reply:
sure, done